### ISSUE TITLE
Wrong fusing operation

### ISSUE THREAD

**Issue by mbrutrifork** [guojiajeremy/Dinomaly#36]
The function for fusing layers calculates the mean:

```python
def fuse_feature(self, feat_list):
        return torch.stack(feat_list, dim=1).mean(dim=1)
```

But the paper shows that it's a sum operation instead.

<img width="1485" height="533" alt="Image" src="https://github.com/user-attachments/assets/30378e12-bfb8-48b1-b08b-fa0d94b8e359" />

Can you confirm?

**Comment by guojiajeremy** [comment]
mean and sum is the same. There is a normalization in calculating cosine similarity.
